docs/shared/cli-install/spm-xcode.mdx

```diff
@@ -4,10 +4,10 @@ To simplify accessing the Codegen CLI, you can run the included `InstallCLI` SPM
 
 This plugin builds the CLI and creates a symbolic link to the executable in your project root.
 
-When using Swift packages through Xcode, right-click on your project in the Xcode file explorer and at the bottom of the menu you will find the `InstallCLI` plugin command. Clicking on this will present a dialog asking for permission for the plugin to write to your project directory.
+If you use Swift packages through Xcode, you can right-click on your project in the Xcode file explorer, revealing an **Install CLI** plugin command. Selecting this command presents a dialog allowing you to grant the plugin "write" access to your project directory.
 
 <img class="screenshot" src="../../source/screenshot/apollo-xcode-plugin.png" alt="Where to find the SPM plugin commands in Xcode" />
 
-After running the installation plugin, a symbolic link to the Codegen CLI named `apollo-ios-cli` is located in your project root folder. You can now run the CLI from the command line with `./apollo-ios-cli`.
+After the plugin installs, it creates a symbolic link to the Codegen CLI (named `apollo-ios-cli`) in your project root folder. You can now run the CLI from the command line with `./apollo-ios-cli`.
 
-> **Note:** Because the `apollo-ios-cli` in your project root is only a symbolic link, it will only work if the compiled CLI exectuable exists. This is generally located in your Xcode Derived Data or the `.build` folder. If these are cleared, you can run the install plugin again to re-build the CLI executable.
+> **Note:** Because the `apollo-ios-cli` in your project root is only a symbolic link, it only works if the compiled CLI executable exists. This is generally located in your Xcode Derived Data or the `.build` folder. If these are cleared, you can run the reinstall plugin to rebuild the CLI executable.
```

Comment:
```suggestion
> **Note:** Because the `apollo-ios-cli` in your project root is only a symbolic link, it only works if the compiled CLI executable exists. This is generally located in your Xcode Derived Data or the `.build` folder. If these are cleared, you can rerun the Install CLI plugin to rebuild the CLI executable.
```